docs/content/guide/external.ngdoc

```diff
@@ -0,0 +1,146 @@
+@ngdoc overview
```

Comment:
Related to the whole file (not this particular line): `guide/external` is a bit vague. Adding `resources` to the name of the file would make it more explicit.